packages/core/src/types/config.ts

```diff
@@ -74,12 +74,33 @@ export type Shims = {
   };
 };
 
+export type JsRedirect = {
+  /**
+   * Whether to automatically redirect the import paths of JavaScript output files.
+   * @defaultValue `true`
+   */
+  path?: boolean;
+  /**
+   * Whether to automatically add the file extension to import paths based on the JavaScript output files.
+   * @defaultValue `true`
+   */
+  extension?: boolean;
+};
+
+// @ts-expect-error TODO: support dts redirect in the future
+type DtsRedirect = {
+  path?: boolean;
+  extension?: boolean;
+};
+
 export type Redirect = {
-  // TODO: support other redirects
-  // alias?: boolean;
+  /** Controls the redirect of the import paths of output JavaScript files. */
```

Comment:
not only import paths?

Reply:
here the paths means the full path, which is used in other places as well.